label_studio/data_manager/api.py

```diff
@@ -224,11 +225,23 @@ def sync_paginate_queryset(self, queryset, request, view=None):
         self.total_annotations = Annotation.objects.filter(task_id__in=queryset, was_cancelled=False).count()
         return super().paginate_queryset(queryset, request, view)
 
+    def paginate_totals_queryset(self, queryset, request, view=None):
+        totals = queryset.values('id', 'total_predictions', 'total_annotations', 'cancelled_annotations').aggregate(
+            total_annotations=Sum('total_annotations') - Sum('cancelled_annotations'),
```

Comment:
It looks like `total_annotations` doesn't include `cancelled_annotations`, so we don't need to subtract them.

```
    total_annotations = models.IntegerField(
        _('total_annotations'),
        default=0,
        db_index=True,
        help_text='Number of total annotations for the current task except cancelled annotations',
    )
```

Reply:
total_annotations=Sum('total_annotations') - Sum('cancelled_annotations')

Reply:
I saw that, but everywhere I saw it appeared to not populate with that notion. Also every single test we have passed with this change 🤔

Reply:
Nope you are right:

```python
def update_tasks_counters(queryset, from_scratch=True):
    """
    Update tasks counters for the passed queryset of Tasks
    :param queryset: Tasks to update queryset
    :param from_scratch: Skip calculated tasks
    :return: Count of updated tasks
    """
    objs = []

    total_annotations = Count('annotations', distinct=True, filter=Q(annotations__was_cancelled=False))
    cancelled_annotations = Count('annotations', distinct=True, filter=Q(annotations__was_cancelled=True))
    total_predictions = Count('predictions', distinct=True)
    ```